src/databricks/labs/ucx/install.py

```diff
@@ -405,6 +442,7 @@ def __init__(
         ws: WorkspaceClient,
         prompts: Prompts,
         verify_timeout: timedelta,
+        product_info: str = PRODUCT_INFO.product_name(),
```

Comment:
it's confusing the reader. product info has to be of `ProductInfo` type, not `str`.